CHANGELOG.md

```diff
@@ -1,3 +1,43 @@
+# v11.0.0b1
```

Comment:
so you're releaseing a1 and b1 together?

Reply:
no. This is copying the a1 release note from the release branch, since we didn't updated the release note in the master branch when we did a1 release.